src/borough/borough.service.spec.ts

```diff
@@ -53,4 +63,23 @@ describe("Borough service unit", () => {
       expect(zoningDistrict).rejects.toThrow(ResourceNotFoundException);
     });
   });
+
+  describe("findCapitalProjectsByBoroughIdCommunityDistrictId", () => {
+    const boroughId = boroughRepositoryMock.checkBoroughByIdMocks[0].id;
+    const communityDistrictId =
+      communityDistrictRepositoryMock.checkCommunityDistrictByIdMocks[0].id;
+    it("service should return a capital projects compliant object", async () => {
+      const capitalProjects =
+        await boroughService.findCapitalProjectsByBoroughIdCommunityDistrictId({
+          boroughId,
+          communityDistrictId,
+        });
+
+      expect(() =>
+        findCapitalProjectsByBoroughIdCommunityDistrictIdQueryResponseSchema.parse(
+          capitalProjects,
+        ),
+      ).not.toThrow();
+    });
+  });
```

Comment:
There is room for better coverage of the behavior of the limit/offset/total/order.

See the city council district service test for an example suite. [findCapitalProjectsByCityCouncilId](https://github.com/NYCPlanning/ae-zoning-api/blob/46e1dbb52ec5b8443dcb3b804578c93391e5f567/src/city-council-district/city-council-district.service.spec.ts#L53)